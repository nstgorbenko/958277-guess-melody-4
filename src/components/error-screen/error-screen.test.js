import React from "react";
import renderer from "react-test-renderer";
import ErrorScreen from "./error-screen.jsx";

describe(`ErrorScreen Component rendering`, () => {
  it(`should render correctly`, () => {
    const tree = renderer
      .create(<ErrorScreen />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
