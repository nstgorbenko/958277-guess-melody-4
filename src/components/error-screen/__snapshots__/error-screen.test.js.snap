// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen Component rendering renders correctly 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Упс!
  </h2>
  <p
    className="result__total result__total--fail"
  >
    Что-то пошло не так, перезагрузите страницу.
  </p>
</section>
`;
